src/hyperlight_guest_bin/src/paging.rs

```diff
@@ -65,64 +87,22 @@ struct MapResponse {
 ///   if previously-unmapped ranges are not being mapped, TLB invalidation may need to be performed afterwards.
 #[instrument(skip_all, parent = Span::current(), level= "Trace")]
 pub unsafe fn map_region(phys_base: u64, virt_base: *mut u8, len: u64) {
```

Comment:
we have pretty minimal test coverage on this from an integration perspective, is there any improvements we could make while we are here or is set of tests we have good enough?

Reply:
I think we need more tests but probably better to do this at the end of these changes since I suspect this function will need to be updated , created #1123 for this